README.rst

```diff
@@ -20,5 +22,6 @@ The DSN format is similar to that of regular Postgres::
     >>> sa.create_engine('redshift+psycopg2://username@host.amazonaws.com:5439/database')
     Engine(redshift+psycopg2://username@host.amazonaws.com:5439/database)
 
-See the docstring for `RedshiftDDLCompiler` in
-`dialect.py <redshift_sqlalchemy/dialect.py>`_ for more detail.
+See the `RedshiftDDLCompiler documentation
+<http://sqlalchemy-redshift.readthedocs.org/en/latest/ddl-compiler.html>`_
+for details on Redshift-specific features the dialect supports.
```

Comment:
Likewise, this link is currently broken on rtfd. This version should work on both GitHub and rtfd.